pypresence/utils.py

```diff
@@ -60,7 +60,7 @@ def get_ipc_path(pipe=None):
         ipc = f"{ipc}{pipe}"
 
     tmp_dir, folder_names = _get_probable_discord_path()
-    if (tmp_dir == None or len(folder_names) == 0):
+    if (tmp_dir is None or len(folder_names) == 0):
```

Comment:
The linter used within the workflow checks gave that it would be better to use an `is` check for None instead of one with `==`. Corrected that within that little followup commit (Electrenator/pypresence@2101b975521484ce526267e9b7519178d3c44f91), among with 1 variable name correction and a typo. That's why that change is there